libtailscale/vpnfacade.go

```diff
@@ -0,0 +1,97 @@
+// Copyright (c) Tailscale Inc & AUTHORS
+// SPDX-License-Identifier: BSD-3-Clause
+
+package libtailscale
+
+import (
+	"sync"
+
+	"tailscale.com/net/dns"
+	"tailscale.com/wgengine/router"
+)
+
+// VPNFacade is an implementation of both wgengine.Router and
+// dns.OSConfiguratof. When ReconfigureVPN is called by the backend, SetBoth
+// gets called.
+type VPNFacade struct {
```

Comment:
This replaces the CallbackRouter from OSS.

Reply:
Sure you don't want to get the cedilla in the name? 🤪